avro/src/schema.rs

```diff
@@ -111,7 +111,7 @@ pub enum Schema {
     /// The underlying type is serialized and deserialized as `Schema::Bytes`
     BigDecimal,
     /// A universally unique identifier, annotating a string.
-    Uuid,
+    Uuid(UuidSchema),
```

Comment:
This is a breaking change, so we need to use `feat!: ...` for the PR title/commit message